invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/PatronDetails/components/ItemsSearch/components/ItemsResultsList/ItemsResultsList.js

```diff
@@ -102,27 +102,27 @@ export class ItemsResultsList extends Component {
     );
   };
 
-  prepareData(data) {
-    return data.hits.map(row => {
-      let serialized = formatter.item.toTable(row);
-      serialized['Actions'] = this.actions(row, row.metadata.status);
-      return pick(serialized, [
-        'ID',
-        'Status',
-        'Medium',
-        'Circulation status',
-        'Actions',
-      ]);
-    });
-  }
-
   render() {
-    const rows = this.prepareData(this.props.results);
-    return rows.length ? (
+    const { results } = this.props;
+    const columns = [
+      { title: '', field: '', formatter: this.viewDetails },
+      { title: 'ID', field: 'metadata.pid' },
+      { title: 'Status', field: 'metadata.status' },
+      { title: 'Medium', field: 'metadata.medium' },
+      { title: 'Circulation status', field: 'metadata.circulation.state' },
+      {
+        title: 'Actions',
+        field: '',
```

Comment:
Would it work if the field was left out from the object here?

Reply:
Yes, but if the title is left out, React is going to throw an error about keys when you iterate on stuff.